adaptor/elasticsearch/clients/v5/writer.go

```diff
@@ -90,10 +91,47 @@ func (w *Writer) Write(msg message.Msg) func(client.Session) (message.Msg, error
 			w.bp.Flush()
 			br = elastic.NewBulkDeleteRequest().Index(w.index).Type(indexType).Id(id)
 		case ops.Insert:
-			br = elastic.NewBulkIndexRequest().Index(w.index).Type(indexType).Id(id).Doc(msg.Data())
+			if _, ok := msg.Data()["elastic_parent"]; ok {
+
+				var parent_id string
+				switch msg.Data()["elastic_parent"].(type) {
```

Comment:
I prefer doing these types of data manipulations outside of the individual adaptors and instead rely on a JS transform function.